src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,21 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+public class AuthenticationService extends UserService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (checkEmailAndPassword(password, email)) {
+            return true;
+        }
```

Comment:
simplify this method. Use `findByEmail` and if `user != null` compare passwords.